mgo_depot.go

```
@@ -109,6 +108,36 @@ func (m *mgoCertDepot) Check(tag *depot.Tag) bool {
 	}
 }
 
+// CheckWithError returns whether the user and data specified by the tag exists
+// as well as an error in the case of an internal error.
+func (m *mgoCertDepot) CheckWithError(tag *depot.Tag) (bool, error) {
+	name, key, err := getNameAndKey(tag)
+	if err != nil {
+		return false, nil
```

Is this supposed to ignore the error?

---

No, good catch.